features/support/api_helpers.rb

```diff
@@ -312,19 +308,19 @@ def create_live_service(framework_slug, lot_slug, supplier_id, role=nil)
   JSON.parse(response.body)['services']
 end
 
-def create_user(user_role, supplier_id=nil)
+def create_user(user_role, custom_user_data={})
   randomString = SecureRandom.hex
   password = ENV["DM_PRODUCTION_#{user_role.upcase.gsub('-', '_')}_USER_PASSWORD"]
 
   user_data = {
-    "emailAddress" => randomString + '@example.gov.uk',
-    "name" => "#{user_role.capitalize} Name #{randomString}",
+    "emailAddress" => custom_user_data['emailAddress'] || custom_user_data['email_address'] || randomString + '@example.gov.uk',
```

Comment:
Why might we have/allow both camel and snake-case versions here?

Reply:
Because to update the email address in the api we use `email_address` but return it in camel case :/